patternfly/patternfly-elements
# WIP: fix: band grid settings

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

* Added variables to the system for standard grid gaps, per breakpoint
* Added an `align-self` variable hook to pfe-card to allow the card to fill or not-fill it's container
* Added a `max-height: fit-content` to pfe-cta to prevent it from getting too fat (tall) in a grid layout.

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. Open the demo page in various browsers to ensure the layout variables are being honored.

#### Browser requirements
Your component should work in all of the following environments:


### Ready-for-merge Checklist



**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**

## TODO
![image](https://user-images.githubusercontent.com/456863/68236839-eebc2200-ffd3-11e9-8197-9bce2d31869a.png)
